tests/iface_loopback_action/iface_loopback_action_helper.py

```diff
@@ -169,16 +169,23 @@ def remove_orig_dut_port_config(duthost, orig_ports_configuration):
     :param duthost: DUT host object
     :param orig_ports_configuration: original ports configuration parameters
     """
+    remove_acl_tables(duthost)
     for _, port_dict in list(orig_ports_configuration.items()):
         port = port_dict['port']
         if port_dict['vlan']:
             remove_dut_vlan_member(duthost, port, port_dict['vlan'])
         elif port_dict['portchannel']:
             remove_dut_portchannel_member(duthost, port, port_dict['portchannel'])
+            output = duthost.shell('show interface portchannel | grep "{} "'.format(port_dict['portchannel']))
+            # Empty portchannels need to be removed before warm/fast reboot
+            if "Ethernet" not in output['stdout']:
```

Comment:
Is it necessary to remove the IP address before removing an empty LAG?

Reply:
Yes, otherwise the warm-reboot will be affected. This is the info from @stepanblyschak when debugging an internal issue([Functional] [fast/warm reboot] [LAG]| Failed to execute fast/warm reboot(RESTARTCHECK failed) after a portchannel is removed) :
```
PortChannel101 still has IP configuration on it:

  "PORTCHANNEL_INTERFACE|PortChannel101": {
  "PORTCHANNEL_INTERFACE|PortChannel101|10.0.0.56/31": {
  "PORTCHANNEL_INTERFACE|PortChannel101|FC00::71/126": {

Therefore, orchagent cannot remove PortChannel101:

May 31 10:07:41.149471 r-leopard-70 NOTICE swss#orchagent: :- doTask: RESTARTCHECK notification for orchagent
May 31 10:07:41.149471 r-leopard-70 NOTICE swss#orchagent: :- doTask: orchagent|NoFreeze:false|SkipPendingTaskCheck:false
May 31 10:07:41.149491 r-leopard-70 ERR swss#orchagent: :- removeLag: Failed to remove ref count 4 LAG PortChannel101
May 31 10:07:41.149537 r-leopard-70 NOTICE swss#orchagent: :- warmRestartCheck: WarmRestart check found pending tasks:
May 31 10:07:41.149566 r-leopard-70 NOTICE swss#orchagent: :- warmRestartCheck:     LAG_TABLE:PortChannel101|DEL
May 31 10:07:41.149566 r-leopard-70 NOTICE swss#orchagent: :- warmRestartCheck: Restart check result: NOT_READY

User needs to remove IP addresses first to remove RIF, then remove the port channel. 

If you enable "yang_config_validation" in DEVICE_METADATA you should have YANG validation for CLI. In that case, the remove command will be rejected.
```